airbyte-workers/src/main/java/io/airbyte/workers/protocols/airbyte/DefaultAirbyteDestination.java

```diff
@@ -122,9 +122,7 @@ public void close() throws IOException {
     LOGGER.debug("Closing destination process");
     WorkerUtils.gentleClose(destinationProcess, 10, TimeUnit.HOURS);
     if (destinationProcess.isAlive() || destinationProcess.exitValue() != 0) {
-      LOGGER.warn(
-          "Destination process might not have shut down correctly. destination process alive: {}, destination process exit value: {}. This warning is normal if the job was cancelled.",
-          destinationProcess.isAlive(), destinationProcess.exitValue());
+      throw new WorkerException("Destination process terminated with exit code " + destinationProcess.exitValue());
```

Comment:
can we change this exception's error message to be similar to the previous one? otherwise it can be noisy + scary for devs and users at a first glance